ui/src/app/job-details/tabs/tabs.component.ts

```diff
@@ -54,6 +55,9 @@ export class JobTabsComponent implements OnInit, OnChanges {
     if (this.hasCallCachedTask() || this.hasScatteredTask()) {
       this.displayedColumns.splice(1, 0, "taskInfoIcons");
     }
+    if (this.tabsPanel) {
+      this.tabWidth = this.tabsPanel._viewContainerRef.element.nativeElement.clientWidth;
```

Comment:
Does this add the responsiveness to the timing panel?

Reply:
It allows the timing diagram component to determine the width to be initially rendered at.  It does not change if the size of the browser window is changed without a reload.

Reply:
no, it's just passed along to the function that builds the timing diagram when the component is loaded